ro_py/users.py

```diff
@@ -56,8 +58,17 @@ async def update(self):
         self.is_banned = user_info["isBanned"]
         self.name = user_info["name"]
         self.display_name = user_info["displayName"]
+        
+        created_date = iso8601.parse_date(user_info["created"])
+        now = datetime.datetime.today()
+        years = now.year - created_date.year
+        months = now.month - created_date.month
+        days = now.day - created_date.day + years * 365 + months * 31
+        self.age = days
```

Comment:
We already have a creation date. Is adding another age really useful here? We expect the user to calculate this themselves.

Reply:
What I can do is remove and use the creation_date already there

Reply:
Removed the created_date and instead used `self.created`